### DISCUSSION TITLE
LiveCD does not load for me

### DISCUSSION THREAD

**Discussion by xybytz** [nerun/remastersys#19] [Questions & Answers]
Hello, I am on Ubuntu 22.04.. When I load the iso in Vmware, it fails to load bluethooth, snap, and unmount the medium. I am not posting this as an issue because I think it is a Me-issue. I am using the CLI version, and typed the command "remastersys dist" to generate.

**Comment by nerun** [comment]
Ok, we can talk here for now. Could you copy/paste your `/home/remastersys/remastersys.log` here? Or a link to it?

**Reply by xybytz** [reply]
Here is a link: https://droptext.cc/47bij

**Reply by xybytz** [reply]
I can try it again doing remastersys dist cdfs first and then remastersys dist iso if you would like.

**Reply by xybytz** [reply]
Ok, tried it again. It works-ish. my password is pac (dont worry its for a vm) and thats not the new password to the new system.

**Reply by nerun** [reply]
When running a live media:
- user and password will be the same: always "live". You can change it in settings.

After install a live media:
- backup mode: user and password will be the ones of the backuped system
- dist mode: user and password will be the ones provided in the installation process.

**Reply by xybytz** [reply]
Weird thing is though that there was no installation process. it was already done. Weird.
